cmake/Jemalloc.cmake

```diff
@@ -1,47 +1,35 @@
-# Source : https://github.com/c9s/r3/blob/2.0/cmake/Modules/FindJemalloc.cmake
-# Licence : MIT license
+# Source : https://github.com/c9s/r3/blob/2.0/cmake/Modules/FindJemalloc.cmake Licence : MIT license
```

Comment:
This is another case where we're pulling in code, but we don't spare it from cmake-format.  We should probably try to be good citizens by keeping the substance of the copied code unmodified.